Ht 21/update GitHub readme about your project (#19)
kalibria/Honda_tracker: README.md

# Honda Tracker - booking app

> The app for booking a car.

Reviewer: ```suggestion
A demo app for booking a family car.
```

Author: changed

## Table of contents

* [General info](#general-info)
* [Technologies](#technologies)
* [Setup](#setup)
* [Illustrations](#illustrations)

## General info

A simple app to track the time when users book Honda car. This application can be useful if several people use the same car.

> It is a responsive design app (mobile and desktop version).

Reviewer: ```suggestion
It is a responsive app (mobile and desktop version).
```

Author: changed

> After logged-in user see a calendar with booking information. User can create, edit, delete a booking, complete a ride, see a booking details, choose settings. All booking information comes from Mongo DB.

Reviewer: ```suggestion
After logging in, the user sees a calendar with booking information. User can create, edit, delete a booking, complete a ride, see booking details, choose settings. All booking information comes from DynamoDB.
```

Author: changed

## Technologies

* React
* Typescript
* CSS: Tailwind
* Forms: Formik
* Intl.DateTimeFormat
* RTK query

> * Postman client for backend requests

Reviewer: Would be nice to export a Postman collection and store the resulting json file in the app

Author: changed

# Honda Tracker - booking app

> A demo app for booking a family car.

Reviewer: ```suggestion
A demo app for booking a family car.
```

```
test
```

> A demo app for booking a family car.

Reviewer: ```suggestion
TEST
```
